src/test/java/core/basesyntax/model/FruitTransactionTest.java

```diff
@@ -0,0 +1,49 @@
+package core.basesyntax.model;
+
+import core.basesyntax.service.OperationHandler;
+import core.basesyntax.strategy.OperationHandlerBalance;
+import core.basesyntax.strategy.OperationHandlerIn;
+import core.basesyntax.strategy.OperationHandlerOut;
+import java.util.Map;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+class FruitTransactionTest {
+    private Map<FruitTransaction.Operation, OperationHandler>
+            correspondenceTable = Map.of(
+            FruitTransaction.Operation.BALANCE, new OperationHandlerBalance(),
+            FruitTransaction.Operation.SUPPLY, new OperationHandlerIn(),
+            FruitTransaction.Operation.RETURN, new OperationHandlerIn(),
+            FruitTransaction.Operation.PURCHASE, new OperationHandlerOut());
+
+    @Test
+    void operationGetByOperationIs_Ok() {
+        FruitTransaction.Operation b = FruitTransaction.Operation.getByCode("b");
```

Comment:
Never use 1-2 letter abbreviations for naming
In a week when you forget code context, you will hate such a decision.
Please recheck all methods
```suggestion
        FruitTransaction.Operation balance = FruitTransaction.Operation.getByCode("b");
```